packages/stylex/package.json

```diff
@@ -34,13 +24,12 @@
   },
   "license": "MIT",
   "scripts": {
-    "prebuild": "gen-types -i src/ -o lib/",
-    "build:cjs": "cross-env BABEL_ENV=cjs babel src/ --out-dir lib/ --copy-files",
-    "build:esm": "cross-env BABEL_ENV=esm babel src/ --out-dir lib/es --out-file-extension .mjs",
-    "postbuild:esm": "rollup -c ./rollup.config.mjs",
+    "prebuild": "rimraf lib && gen-types -i src/ -o lib/cjs && cp -r lib/cjs lib/es",
+    "build:cjs": "cross-env BABEL_ENV=cjs rollup -c ./rollup.config.mjs",
+    "build:esm": "cross-env BABEL_ENV=esm rollup -c ./rollup.config.mjs",
     "build": "npm run build:cjs && npm run build:esm",
-    "build-haste": "rewrite-imports -i src/ -o lib/",
-    "test": "jest"
+    "build-haste": "rimraf lib && rewrite-imports -i src/ -o lib/",
```

Comment:
During the internal sync, we run the esm AND cjs builds for some reason (as this package is a dependency of the babel-plugin build). Later, we run the haste build but don't seem to use the esm/cjs builds, so I'm assuming it's safe to clean out the directory before this build.